package.json

```diff
@@ -55,7 +55,8 @@
     "@commitlint/config-conventional": "^17.0.3",
     "@types/node": "^18.6.4",
     "husky": "^8.0.0",
-    "semantic-release": "^19.0.3"
+    "semantic-release": "^19.0.3",
+    "typescript": "^4.6.4"
```

Comment:
Without this the CI was failing with a mismatch between node and dom setup around AbortSignal. This mismatch seems to be introduced with [TS 4.9](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62782). Which is what the CI probably upgraded to. Locally with TS 4.6 I couldn't reproduce the problem.